src/main/java/Duke.java

```diff
@@ -1,10 +1,138 @@
+import java.util.Scanner;
+import java.util.ArrayList;
 public class Duke {
+    private static final String LINE = "____________________________________________________________";
+    private static ArrayList<Task> list = new ArrayList<Task>();
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        greet();
+        while (true) {
+            Scanner in = new Scanner(System.in);
+            String input = in.nextLine();
+            if(input.equalsIgnoreCase("bye")) {
+                bye();
+                break;
+            }
+            processInput(input);
```

Comment:
I like how you grouped the processing of all the different commands into one method call, it makes the code cleaner and more readable.